<ui version="4.0" >
 <class>ColumnDialog</class>
 <widget class="QWidget" name="ColumnDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gbGeneral" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <item row="2" column="0" >
       <widget class="QLabel" name="lFormat" >
        <property name="text" >
         <string>Format :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lLabel" >
        <property name="text" >
         <string>Label :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="lType" >
        <property name="text" >
         <string>Type :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leLabel" />
      </item>
      <item row="2" column="1" >
       <widget class="KComboBox" name="cbFormat" />
      </item>
      <item row="4" column="1" >
       <widget class="KComboBox" name="cbType" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>selectFile()</slot>
  <slot>updateFileType()</slot>
  <slot>fileInfoDialog()</slot>
 </slots>
</ui>
